### PR TITLE
feat: pin exact dependency versions 

### PR DESCRIPTION
**Pull request by dim-daskalov**
Pin the dependency version and replace the 'tweetnacl' package with @noble/curves, this PR contains the commits from 'update-naj-to-v4' branch.

### CODE REVIEW

File: examples/angular/src/app/pages/wallet-selector-export/wallet-selector-export.component.ts [hunk 1]
```diff
@@ -81,6 +82,7 @@ export class WalletSelectorExportComponent implements OnInit {
         setupRamperWallet(),
         setupNearMobileWallet(),
         setupMintbaseWallet({ contractId: "guest-book.testnet" }),
+        setupBitteWallet({ contractId: "guest-book.testnet" }),
```
Comment (pivanov) [inline examples/angular/src/app/pages/wallet-selector-export/wallet-selector-export.component.ts]: We can use `CONTRACT_ID` constant from `./examples/angular/src/constants.ts`

File: examples/angular/src/app/pages/wallet-selector/wallet-selector.component.ts [hunk 2]
```diff
@@ -95,6 +96,7 @@ export class WalletSelectorComponent implements OnInit {
         setupRamperWallet(),
         setupNearMobileWallet(),
         setupMintbaseWallet({ contractId: "guest-book.testnet" }),
+        setupBitteWallet({ contractId: "guest-book.testnet" }),
```
Comment (pivanov) [inline examples/angular/src/app/pages/wallet-selector/wallet-selector.component.ts]: same here

File: examples/react/contexts/WalletSelectorContext.tsx [hunk 3]
```diff
@@ -97,6 +98,7 @@ export const WalletSelectorContextProvider: React.FC<{
         }),
         setupNearMobileWallet(),
         setupMintbaseWallet({ contractId: "guest-book.testnet" }),
+        setupBitteWallet({ contractId: "guest-book.testnet" }),
```
Comment (pivanov) [inline examples/react/contexts/WalletSelectorContext.tsx]: same here

File: examples/react/contexts/WalletSelectorExportContext.tsx [hunk 4]
```diff
@@ -71,6 +72,7 @@ export const ExportAccountSelectorContextProvider: React.FC<{
         setupNearFi(),
         setupRamperWallet(),
         setupMintbaseWallet({ contractId: "guest-book.testnet" }),
+        setupBitteWallet({ contractId: "guest-book.testnet" }),
```
Comment (pivanov) [inline examples/react/contexts/WalletSelectorExportContext.tsx]: same here

**Review by pivanov — APPROVED** [review]
r+
Overall looks good ... we just need to think about merging and fixing things in one PR

File: examples/angular/src/app/pages/wallet-selector-export/wallet-selector-export.component.ts (hunk 1 above)
Comment (dim-daskalov) [reply examples/angular/src/app/pages/wallet-selector-export/wallet-selector-export.component.ts]: Yep, thanks for mentioning it.

**Comment by pivanov** [comment]
close in favor of [PR#2](https://github.com/LimeChain/wallet-selector/pull/2)

**Pull request closed without merging**